release-notes/DxMs/DataAPI_change_log.md

```diff
@@ -7,6 +7,59 @@ uid: DataAPI_change_log
 > [!IMPORTANT]
 > At present, the Data API feature is only available in preview, if the soft-launch option *DataAPI* is enabled. For more information, see [Getting started with the Data Sources module](xref:Data_Sources_Setup).
 
+### 29 April - Enhancement - DataAPI 1.2.0 - Improved performance when creating new connectors
+
+The ​Data API now uses an updated version of the [Skyline.DataMiner.CICD.Models.Protocol](https://www.nuget.org/packages/Skyline.DataMiner.CICD.Models.Protocol) NuGet package, which improves performance when creating a new connector with many standalone parameters.
+
+### 29 April - Enhancement - DataAPI 1.2.0 - Removed copyright from auto-generated connectors
+
+The auto-generated connectors generated by the Data API no longer contain the copyright message. Instead, these just contain the following line, where the timestamp indicates the UTC timestamp of when the connector was generated:
+
+```csharp
+<!--This protocol is auto-generated by the DataMiner Data API at 2024-04-12T05:43:30.823Z.-->
+```
+
+### 29 April - Fix - DataAPI 1.2.0 - All active elements updated with same identifier
+
+When a user tries to push data to the Data API, and there are stopped elements with the specified identifier, the Data API will now update all active elements with that identifier.
```

Comment:
It's not really clear what the issue is that is fixed here. Could you rephrase this so this will be more clear for a user who may have encountered this issue?

Reply:
updated description